src/components/RequestsTable/FilterRequests.tsx

```diff
@@ -0,0 +1,334 @@
+                .filter((obj) => {
+                  return obj.column === "series";
+                })[0]
+                ?.filter.toString()}
+              maxLength={4}
+            />
+          </Field>
+          <hr />
+          <Field label="Grade">
+            <Combobox
+              name="grade"
+              clearable
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "grade";
+                })[0]
+                ?.filter.toString()}
+            >
+              {ACQGRADES.map((grade) => (
+                <Option key={grade} value={grade}>
+                  {grade}
+                </Option>
+              ))}
+              {GENERALGRADES.map((grade) => (
+                <Option key={grade} value={grade}>
+                  {grade}
+                </Option>
+              ))}
+            </Combobox>
+          </Field>
+          <hr />
+          <Field label="Office Symbol">
+            <Input
+              name="officeSymbol"
+              type="search"
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "officeSymbol";
+                })[0]
+                ?.filter.toString()}
+            />
+          </Field>
+          <hr />
+          <Field label="Created By">
+            <Input
+              name="Author"
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "Author";
+                })[0]
+                ?.filter.toString()}
+            />
+          </Field>
+          <hr />
+          <Field label="Stage">
+            <Combobox
+              name="stage"
+              clearable
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "stage";
+                })[0]
+                ?.filter.toString()}
+            >
+              {STAGES.map((stage) => (
+                <Option key={stage.key} text={stage.key} value={stage.key}>
+                  {stage.text}
+                </Option>
+              ))}
+            </Combobox>
+          </Field>
+          <hr />
+          <Field label="Created After">
+            <DatePicker
+              name="afterDate"
+              value={afterDate ? new Date(afterDate) : undefined}
+              formatDate={(date?: Date) => {
+                return !date
+                  ? ""
+                  : date.getFullYear() +
+                      "-" +
+                      (date.getUTCMonth() + 1) +
+                      "-" +
+                      date.getDate();
+              }}
+            />
+          </Field>
+          <hr />
+          <Field label="Created Before">
+            <DatePicker
+              name="beforeDate"
+              value={beforeDate ? new Date(beforeDate) : undefined}
+              formatDate={(date?: Date) => {
+                return !date
```

Comment:
Not a big fan of the single digit date/month -- I like the 0 prefix

Reply:
Looks good to me.  Like the consistency, and for some reason I accept no leading zeroes in this format, just really don't like it the other way for some reason.